float32 rho
float32 theta
int8 imageCenter_x
int8 imageCenter_y
int8 projection_x
int8 projection_y
float32 perpendicular_distance
